aries_cloudagent/protocols/present_proof/dif/pres_exch_handler.py

```diff
@@ -1134,10 +1138,14 @@ async def create_vp(
             if not issuer_id and len(filtered_creds_list) == 0:
                 vp = await create_presentation(credentials=applicable_creds_list)
                 vp["presentation_submission"] = submission_property.serialize()
+                if self.proof_type is BbsBlsSignature2020.signature_type:
+                    vp["@context"].append("https://w3id.org/security/bbs/v1")
```

Comment:
Suggest to use the constant from the `vc/ld_proof/constants` instead of plain string